rate-my-noodles/rate_my_noodles.py

```diff
@@ -22,11 +22,23 @@
     "ramen_noodles",
     con=engine
 )
+print("Brand?")
 brand = input()
+print("Variety?")
 variety = input()
```

Comment:
We can combine 25-26 and 27-28, with the input containing the prompt.
brand = input('Brand? ')
variety = input('Variety? ')